software/bewcloud.yml

```
@@ -0,0 +1,17 @@
+name: "bewCloud"
+website_url: "https://bewcloud.com"
+source_code_url: "https://github.com/bewcloud/bewcloud"
+description: "A simpler alternative to Nextcloud and ownCloud's RSS Reader, File Sharing + Sync, Notes, and Photos."
+licenses:
+  - AGPL-3.0
+platforms:
+  - Deno
+  - Nodejs
+  - Docker
```

```suggestion
platforms:
  - Docker
```

Docker is the only supported deployment method

>  Values for platform are the main server-side requirements for the software. Don't include frameworks or specific dialects.